.github/workflows/rebase.yml

```
@@ -0,0 +1,18 @@
+# https://github.com/marketplace/actions/automatic-rebase  (https://github.com/cirrus-actions/rebase)
+on:
+  issue_comment:
+    types: [created]
+name: Automatic Rebase
+jobs:
+  rebase:
+    name: Rebase
+    if: github.event.issue.pull_request != '' && startsWith(github.event.comment.body, '/rebase')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@master
+      with:
+        fetch-depth: 0
+    - name: Automatic Rebase
+      uses: cirrus-actions/rebase@1.3.1
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Personally, I prefer `env` to be extracted to the top of the workflow, so it's immediately clear what env vars are needed for the job.

```suggestion
name: Automatic Rebase

on:
  issue_comment:
    types: [created]

env:
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

jobs:
  rebase:
    name: Rebase
    if: github.event.issue.pull_request != '' && startsWith(github.event.comment.body, '/rebase')
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
      with:
        fetch-depth: 0
    - name: Automatic Rebase
      uses: cirrus-actions/rebase@1.3.1
```

---

Ohh..So it will still work like that? I did not know that. I will commit your suggestions @timbuckley